doc/Developing.md

```diff
@@ -44,14 +44,29 @@ For Windows instructions [click here](#building-on-windows).
      - [Root privileges errors](#root-privileges-errors)
 
 ## Prerequisites
+
  - Node.js `>= 10.2.0`.
    - Preferably, **use** version `10.15.3`, it has the [active LTS](https://github.com/nodejs/Release).
    - Node.js `11.x` is untested.
  - [Yarn package manager](https://yarnpkg.com/en/docs/install) v1.7.0
  - git (If you would like to use the Git-extension too, you will need to have git version 2.11.0 or higher.)
 
-[nvm](https://github.com/creationix/nvm) is recommended to easily switch between
-Node.js versions.
+Some additional tools and libraries are needed depending on your platform:
+
+- Linux
+  - [make](https://www.gnu.org/software/make/)
+  - [gcc](https://gcc.gnu.org/) (or another compiling toolchain)
+  - [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/)
+  - Dependencies for `native-keymap` node native extension:
```

Comment:
We added links for other dependencies / tools, should we not have added a link here as well?
https://www.npmjs.com/package/native-keymap

Reply:
It is something that is being install on their behalf by Theia, unlike the rest. But it could have been a link indeed.